lab_1/main.py

```diff
@@ -12,9 +24,23 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    tokens = []
+    if type(text) == str:
+        text = text.lower()
+        for symbol in text:
+            if symbol in '''1234567890-=!@#$%^&*()_+`~[]\{}|;':",./<>?''':
```

Comment:
No need for triple-quote since it is one line.

Reply:
I still decided to go back to the regular season. But! I also have single and double quotes among the symbol -- so I used triple quotes (just want to know for the future)

Reply:
sooooooooooooooooooooooooooo